addons/project/tests/test_project_subtask_ui_flow.py

```diff
@@ -0,0 +1,22 @@
+from odoo import Command
+from odoo.tests import HttpCase, tagged, Form
+
+from .test_project_base import TestProjectCommon
+
+@tagged('-at_install', 'post_install')
+class TestProjectSubtaskUIFlow(HttpCase, TestProjectCommon):
+    def test_invisible_subtask_became_visible_when_converted_to_task(self):
+        task = self.env['project.task'].create({
+            'name': 'Parent task',
+            'project_id': self.project_goats.id,
+            'child_ids': [Command.create({ 'name': 'Sub-task invisible', 'project_id': self.project_goats.id, })],
+        })
+        invisible_subtask, = task.child_ids
```

Comment:
Since there is only one subtask, we could do that:
```suggestion
        invisible_subtask = task.child_ids
```